scripts/ASTImportTest.sh

```diff
@@ -103,29 +103,29 @@ do
         done
     elif [ ${SPLITSOURCES_RC} == 1 ]
     then
-        testImportExportEquivalence $solfile
+        testImportExportEquivalence "$solfile"
     elif [ ${SPLITSOURCES_RC} == 2 ]
     then
         # The script will exit with return code 2, if an UnicodeDecodeError occurred.
         # This is the case if e.g. some tests are using invalid utf-8 sequences. We will ignore
         # these errors, but print the actual output of the script.
         echo -e "\n${OUTPUT}\n"
```

Comment:
Quotes inside quotes?

Reply:
I don't understand. I don't see any quotes inside quotes here and it does not look like a situation where nested quotes would be needed. What exactly do you mean?

Reply:
I should have said shouldn't there be quotes inside the existing quote like so `"\n"${OUTPUT}"\n" but I suppose this is not required :-)

Reply:
That would quote only the the `\n` characters though. The first quote would end just before `${OUTPUT}` and the second one would start immediately after it. It's actually one of the errors that shellcheck warns about :)